src/traits.d

```diff
@@ -522,6 +546,15 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
     {
         return isFuncX(f => !f.needThis() && !f.isNested());
     }
+    if (e.ident == Id.isModule)
+    {
+        //TODO: can we handle the case when isPkgMod == PKGunknown?
```

Comment:
That could work. The tricky bit is that since __traits can get either an Import or a Package, we'd probably need to put the method in the Package class.